clang/test/Driver/sycl-device-lib-win.cpp

```diff
@@ -1,7 +1,7 @@
 ///
 /// Perform several driver tests for SYCL device libraries on Windows
 ///
-// REQUIRES: clang-driver, windows
+// UNSUPPORTED: system-linux
```

Comment:
Deleted code seems to express the limitation more precisely. Is it supposed to work on Darwin, Fuchsia or FreeBSD?

Reply:
Hi, @bader 
You are correct, this lit test should be only for windows.
Thanks very much.